bears/c_languages/CPPCleanBear.py

```diff
@@ -22,5 +23,12 @@ class CPPCleanBear:
     CAN_DETECT = {'Smell', 'Unused Code', 'Security'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return filename,
+    def create_arguments(filename,
+                         file,
+                         config_file,
+                         include_paths:
+                         typed_list(str)=()):
+        include = ' '.join('--include-path='+part.strip()
+                           for part in include_paths)
```

Comment:
is joining with spaces correct? Or does cppclean need a different separator like comma? (I'm asking as bash/shells don't support spaces in a single argument except when quoting. Although executables in linter are not run in shell and you don't have to quote arguments in here, when writing this equivalently in bash using spaces is rather unusual when specifying multiple arguments like here)

Reply:
nvm misread code.
However, as stated, this is not shell, so you have to put each of those `--include-path=...` into a list :)
Also please add a test using multiple include paths, to test support for this + to avoid the error you already have here when specifying multiple include-paths

Reply:
@Makman2 Yeah I get the problem but am not so clear as to how to solve it. I want each of those --include-path=... seperated with a space, for e.g. `cppclean --include-path=path1 --include-path=path2`. So what you are suggesting is to use a list instead of a typed_list? I would still have to separate them with a space

Reply:
Effectively those arguments are concatted again by a space.

Linter doesn't use a shell. Without a shell arguments get passed inside a sequence. Actually when you write a command into a shell like bash, this happens inside:
```
myprogram arg1 "some other arg"
```
gets parsed to something like this internally:
```python
['myprogram', 'arg1', 'some other arg']
```
and is passed to the program `myprogram`. Now that we don't use a shell, we have to do that argument splitting manually inside a bear (which is more error-proof and easier).

Reply:
@Makman2 Thank you so much for taking the time to explain this to me :) My latest PR is working perfectly when including multiple include-paths and I have appended the test to use multiple include-paths. When you have time  would be great if you could review the changes :)